### PR TITLE
chore/Setup a mono-repo

### PR DESCRIPTION
**Pull request by Navneet-Sahota**
close#1

### CODE REVIEW

**Comment by Navneet-Sahota** [comment]
Hey @nimish-gupta, this is the first PR.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+jspm_packages
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+# Lock files
+yarn.lock
```
Comment (nimish-gupta) [inline .gitignore]: why this?

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (nimish-gupta) [inline client/public/index.html]: title?

File: .gitignore (hunk 1 above)
Comment (Navneet-Sahota) [reply .gitignore]: Wasn't sure if I should commit `yarn.lock` and `package-lock.json` because we already had `package.json`

Comment (Navneet-Sahota) [reply .gitignore]: Will remove lock files them from `.gitignore` in next commit. Will add in next commit

File: client/public/index.html (hunk 2 above)
Comment (Navneet-Sahota) [reply client/public/index.html]: Forgot to do add this. Will add in next commit

**Pull request merged**